cluster/kubernetes/resourcekinds.go

```diff
@@ -509,7 +509,7 @@ func (hr *helmReleaseKind) getWorkloads(ctx context.Context, c *Cluster, namespa
 	if err := ctx.Err(); err != nil {
 		return nil, err
 	}
-	names := make(map[string]bool, 0)
+	names := make(map[string]bool)
```

Comment:
The second argument is only the [initial allocation](https://golang.org/ref/spec#Making_slices_maps_and_channels)
> make(T, n) map of type T with initial space for approximately n elements

If you add more elements the runtime takes care of allocating more space. So the 0 is likely a no-op but shouldn't hurt to be there.